File: frontend-app/src/core/model/dto/response/MonitorServerResponse.ts
import { DEFAULT_MONITOR_SERVER_INFO_OBJ, type MonitorServerInfoResponse } from "./MonitorServerInfoResponse";

export interface MonitorServerResponse {
    id : number;
    host : string;
    info : MonitorServerInfoResponse;
    ativo: boolean;
}

export const DEFAULT_MONITOR_SERVER_OBJ : MonitorServerResponse = {
    id: 0,
    host: '',
    info: DEFAULT_MONITOR_SERVER_INFO_OBJ,
    ativo: false
};

File: main-api/src/main/java/com/italomonitor/main/dto/response/MonitorServerResponse.java
package com.italomonitor.main.dto.response;

import com.italomonitor.main.dto.integration.response.MonitorServerInfoResponse;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@NoArgsConstructor
@AllArgsConstructor
@Getter
@Setter
@Builder
public class MonitorServerResponse {

	private Long id;
	
	private String host;
	
	private boolean ativo;

	private MonitorServerInfoResponse info;
		
}
